about.css

```diff
@@ -0,0 +1,445 @@
+    flex-direction: column;
+    align-items: center;
+    border-top: 1px solid rgb(196, 200, 196);
+    padding-top: 20px;
+  }
+
+  .par-1 {
+    width: 80%;
+    margin: 10px;
+  }
+
+  .img-container {
+    position: relative;
+    width: 70%;
+    margin: 10px;
+  }
+
+  .container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .overlay {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 200px;
+    background-color: rgb(236, 82, 66, 0.5);
+    flex-direction: column;
+    justify-content: center;
+    color: #fff;
+    display: flex;
+    align-items: center;
+    gap: 10px;
+  }
+
+  .gr {
+    width: 100%;
+    height: 200px;
+  }
+
+  .sponsor {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 30px;
+    margin-top: 5%;
+    background-color: #414246;
+    width: 100%;
+    height: 200px;
+    color: #fff;
+  }
+
+  .spon-brands {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-wrap: wrap;
+  }
+
+  .spon-img {
+    width: 115px;
+  }
+
+  .hide-mobile {
+    display: none;
+  }
+}
+
+@media (min-width: 768px) {
+  header {
+    height: 100%;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    overflow: hidden;
+  }
+
+  header::before {
+    content: '';
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    opacity: 0.2;
+    z-index: -1;
+    background-image: url('./img/ab-back\ -\ desk.png');
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-color: #fff;
+    padding-top: 50px;
+  }
+
+  .list-1 {
+    display: flex;
+    background-color: #000b;
+    justify-content: end;
+    color: white;
+    padding-right: 250px;
+    height: 25px;
+  }
+
+  ul {
+    list-style: none;
+  }
+
+  .list-1 ul {
+    display: flex;
+    align-items: center;
+    gap: 30px;
+  }
+
+  a {
+    text-decoration: none;
+  }
+
+  li a {
+    color: #fff;
+  }
+
+  .mobile-menu {
+    display: none;
+  }
+
+  .nav {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    gap: 40px;
+    background-color: #fff;
+    padding: 0 150px 0 150px;
+  }
+
+  .logo {
+    width: 70px;
+    height: 50px;
+  }
+
+  .nav ul {
+    display: flex;
+    align-items: center;
+    gap: 30px;
+  }
+
+  .list-h4 {
+    color: #ec5242;
+    border: 4px solid #ec5242;
+    padding: 5px;
+  }
+
+  .nav a:hover {
+    color: #ec5242;
+  }
+
+  .description {
+    display: flex;
+    flex-direction: column;
+    width: 800px;
+    margin: 100px 200px;
+    text-align: left;
+  }
+
+  .desc-header {
+    font-size: 25px;
+    color: #f00;
+  }
+
+  .description h1 {
+    font-size: 250%;
+    font-weight: 900;
+    color: #f00;
+    line-height: 55px;
+  }
+
+  .p-1 {
+    width: 500px;
+    background-color: #dddddded;
```

Comment:
- **[OPTIONAL]** I marked this requirement as OPTIONAL because it wasn't required but the last CR, but is a project requirement to follow the instruction from the original design. Please 🙏🏽 replace this color `#dddddded` for `#d3d3d3`